images/etcd.bzl

```diff
@@ -0,0 +1,11 @@
+# Returns the supported arch & versions of etcd
+# This avoids repeating the etcd versions in multiple places,
+# and also avoids problems such as 3.1.12 not being available on arm
+def supported_etcd_arch_and_version():
+  return [
+    [ "amd64", "3.1.12" ],
```

Comment:
Can we add ("amd64", "2.2.1") ... I know we don't want to support 2.2.1 for much longer (if at all) but it would make this PR strictly additive.

Of course I may be forgetting why we removed it!

(And no need to support on arm64, there's no user base on arm64/2.2.1 that we need to cater to)

Reply:
I will try to do this, but will not be able to test.
I removed it purely because of build issues. v2.2.1 is built from source and and my Bazel version doesn't seem to like it.

Reply:
It is ok to skip 2.2.1 here as we build the layer differently. The latest commit should fix this.

Reply:
Ah - makes sense, thank you!

Reply:
😄